docs/source/en/guides/download.md

```diff
@@ -126,42 +126,25 @@ files except `vocab.json`.
-    will still be used to check whether the file is already cached or not. If already cached, the file is **duplicated**
-    from the cache (i.e. saves bandwidth but increases disk usage). If the file is not already cached, it will be
-    downloaded and moved directly to the local dir. This means that if you need to reuse it somewhere else later, it
-    will be **re-downloaded**.
-
-Here is a table that summarizes the different options to help you choose the parameters that best suit your use case.
-
-<!-- Generated with https://www.tablesgenerator.com/markdown_tables -->
-| Parameters | File already cached | Returned path | Can read path? | Can save to path? | Optimized bandwidth | Optimized disk usage |
-|---|:---:|:---:|:---:|:---:|:---:|:---:|
-| `local_dir=None` |  | symlink in cache | ✅ | ❌<br>_(save would corrupt the cache)_ | ✅ | ✅ |
-| `local_dir="path/to/folder"`<br>`local_dir_use_symlinks="auto"` |  | file or symlink in folder | ✅ | ✅ _(for small files)_ <br> ⚠️ _(for big files do not resolve path before saving)_ | ✅ | ✅ |
-| `local_dir="path/to/folder"`<br>`local_dir_use_symlinks=True` |  | symlink in folder | ✅ | ⚠️<br>_(do not resolve path before saving)_ | ✅ | ✅ |
-| `local_dir="path/to/folder"`<br>`local_dir_use_symlinks=False` | No | file in folder | ✅ | ✅ | ❌<br>_(if re-run, file is re-downloaded)_ | ⚠️<br>(multiple copies if ran in multiple folders) |
-| `local_dir="path/to/folder"`<br>`local_dir_use_symlinks=False` | Yes | file in folder | ✅ | ✅ | ⚠️<br>_(file has to be cached first)_ | ❌<br>_(file is duplicated)_ |
-
-**Note:** if you are on a Windows machine, you need to enable developer mode or run `huggingface_hub` as admin to enable
-symlinks. Check out the [cache limitations](../guides/manage-cache#limitations) section for more details.
+## Download file(s) to a local folder
+
+By default, we recommend using the [cache system](./manage-cache) to download files from the Hub. You can specify a custom cache location using the `cache_dir` parameter in [`hf_hub_download`] and [`snapshot_download`], or by setting the [`HF_HOME`](../package_reference/environment_variables#hf_home) environment variable.
+
+However, if you need to download files to a specific folder, you can pass a `local_dir` parameter to the download function. This is useful to get a workflow closer to what `git` commands offer. The downloaded files will maintain their original file structure within the specified folder. For example, if `filename="data/train.csv"` and `local_dir="path/to/folder"`, the resulting filepath will be `"path/to/folder/data/train.csv"`.
+
+Note that a `.huggingface/` folder will be created at the root of your local directory, containing metadata about the downloaded files. This prevents re-downloading files if you re-run your script. While this mechanism is not as robust as the main cache-system, it's optimized for regularly pulling the latest version of a repository.
+
+<Tip>
```

Comment:
I would leave this as regular text since we want the user to be aware of what happens if they delete the folder. A tip may suggest to the user that they can skip or gloss over this section which we don't necessarily want. Plus its nice to not have two `Tip`'s in a row :)